metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/api/external/model/TimeSpan.java

```diff
@@ -8,40 +8,40 @@
 import javax.xml.bind.annotation.XmlRootElement;
 
 /**
- * Timespan model class.
+ * TimeSpan model class.
  */
-@XmlRootElement(namespace = "http://www.europeana.eu/schemas/edm/", name = "Timespan")
+@XmlRootElement(namespace = "http://www.europeana.eu/schemas/edm/", name = "TimeSpan")
 @XmlAccessorType(XmlAccessType.FIELD)
-public class Timespan extends EnrichmentBase {
+public class TimeSpan extends EnrichmentBase {
 
   @XmlElement(name = "isPartOf", namespace = "http://purl.org/dc/terms/")
-  private Part isPartOf;
+  private List<LabelResource> isPartOf;
   @XmlElement(name = "hasPart", namespace = "http://purl.org/dc/terms/")
-  private List<Part> hasPartsList = new ArrayList<>();
+  private List<LabelResource> hasPartsList = new ArrayList<>();
   @XmlElement(name = "sameAs", namespace = "http://www.w3.org/2002/07/owl#")
   private List<Part> sameAs = new ArrayList<>();
```

Comment:
Regarding the parts. I think that Part and Resource are the exact same (both are equivalent to rdf:ResourceType) and one of them should go. So I guess the question is whether I should do it now. I decided against it because eventually we should abandon these model classes altogether. I decided just to correct, not to improve. However, it wouldn't be a lot of work. What do you think?

The sameAs property I have not changed it because I didn't need to. It is of type rdf:ResourceType. The hasPartsList however (the name is confusing) should change because it is not a resource, but a resource or literal (rdf:ResourceOrLiteralType). So I corrected this.

Reply:
Yeah I was wondering why the discrimination between those fields..
Anyway if you think we should work on this specific issue, maybe create a ticket?